app/_assets/stylesheets/pages/landing-page.less

```diff
@@ -377,7 +377,7 @@ header.navbar {
     .cards-container {
       display: grid;
       grid-gap: 24px;
-      grid-template-columns: 1fr 1fr 1fr;
+      grid-template-columns: 1fr 1fr;
```

Comment:
I think I prefer the 3 tiles. Would a row of 4 and a row of 3 work? I tried it, and I feel like the top and bottom tile padding gets a bit weird. Maybe the text is too scrunched?